src/Symfony/Component/Config/Tests/Definition/BaseNodeTest.php

```diff
@@ -36,7 +36,31 @@ public function testGetPathForChildNode(string $expected, array $params)
             }
         }
 
-        $node = $this->getMockForAbstractClass(BaseNode::class, $constructorArgs);
+        $node = new class(...$constructorArgs) extends BaseNode {
+            protected function validateType($value)
```

Comment:
Can you please add native return types to all of your new anonymous classes? That'll make the merge-up easier.

Reply:
Added where possible, taking into account that most of the method return `mixed` which is not available in PHP 7.2